<launch>
  <node pkg="intellwheels_rl" type="robot2_dqn.py" name="robot2_dqn" output="screen" >
    <param name="train_mode" value="False" />
    <param name="start_episode" value="100" />    
    <param name="total_episodes" value="200" />
  </node>
</launch>
  